Makefile

```diff
@@ -1,3 +1,21 @@
+.PHONY: \
+	clean \
+	functest \
+	govet \
+	unit-test
+
+clean:
+	@echo "cleaning previous outputs"
+	hack/clean.sh
+
+build-functest:
+	@echo "Building functional tests"
+	hack/build-functest.sh
+
+functest: build-functest
```

Comment:
Call it func-test, similar to unit-test.

Reply:
done